package.json

```diff
@@ -2,7 +2,8 @@
   "scripts": {
     "test": "jest",
     "generate_rule": "node scripts/generate_rule.js",
-    "bulk-update": "node scripts/bulk_update.js"
+    "bulk-update": "node scripts/bulk_update.js",
+    "validate-rule-urls": "node scripts/validate_rule_urls.js"
```

Comment:
At this stage, I don't think it makes sense to add this to the CI / general validation since it's a naïve process (trying to visit each URL) so quite slow and subject to the whim of website availability, and we'd also need some sort of URL "ignore" list for cases like e.g. `www.example.com` in the remediation messages